### PR TITLE
Run the test suite with rails 5.0

### PR DESCRIPTION
**Pull request by orien**
We should test this library against the newly released Rails 5.0.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,24 +9,27 @@ script:
   - ruby script/jack_hammer -t 2000
 matrix:
   include:
-    - rvm: 2.1.8
+    - rvm: 2.2.5
       gemfile: spec/support/gemfiles/Gemfile.rails-3.2.x
       env: DB=mysql
-    - rvm: 2.1.8
+    - rvm: 2.1.9
```
Comment (spickermann) [inline .travis.yml]: Shouldn't this be [`Ruby 2.1.10`](https://www.ruby-lang.org/en/news/2016/04/01/ruby-2-1-10-released/)?

File: .travis.yml [hunk 2]
```diff
@@ -9,24 +9,27 @@ script:
...
       gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
       env: DB=mysql
-    - rvm: 2.2.4
-      gemfile: spec/support/gemfiles/Gemfile.rails-4.2.x
+    - rvm: 2.2.5
+      gemfile: spec/support/gemfiles/Gemfile.rails-5.0.x
       env: DB=mysql
-    - rvm: 2.3.0
+    - rvm: 2.3.1
       gemfile: spec/support/gemfiles/Gemfile.rails-4.1.x
```
Comment (spickermann) [inline .travis.yml]: With the release of Rails 5.0 [Rails 4.1](http://guides.rubyonrails.org/maintenance_policy.html) and below will essentially be unsupported. Does it still make sense to spec against them?

**Comment by spickermann** [comment]
Thanks for ensuring that this gem supports Rails 5.0.

:+1:

File: .travis.yml (hunk 2 above)
Comment (orien) [inline .travis.yml]: Yes, we should remove them eventually. But for now let's leave them in as a favour to all the companies running large monolithic applications on legacy Rails versions and having trouble upgrading. 😉

**Comment by jiexinhuang** [comment]
Cool stuff 👍

File: .travis.yml (hunk 2 above)
Comment (spickermann) [inline .travis.yml]: You are right. That is a nice touch. ;-)

**Pull request merged**